eZ/Publish/Core/REST/Common/FieldTypeProcessor/RelationProcessor.php

```diff
@@ -31,4 +31,17 @@ public function postProcessValueHash($outgoingValueHash)
 
         return $outgoingValueHash;
     }
+
+    public function postProcessFieldSettingsHash($outgoingSettingsHash)
+    {
+        $outgoingSettingsHash = parent::postProcessFieldSettingsHash($outgoingSettingsHash);
+
+        if (isset($outgoingSettingsHash['selectionRoot'])) {
```

Comment:
`isset` does not seem to be enough, it seems like `selectionRoot` can be set with `''`
I got the following output with your patch https://gist.github.com/dpobel/f13327b30dadaed93c2cff7e449f8934